pages/index.tsx

```diff
@@ -20,6 +21,8 @@ export interface HomeProps {
 
 export default function Home({ claimData, loading }: HomeProps): ReactElement {
   const { t } = useTranslation('common')
+  const router = useRouter()
+  const mobile = router.query?.from === 'uiom'
```

Comment:
not sure if this is the right place, but it'd be nice to have a meta-comment to clarify that there's the main/default UIO website, vs. "UIO Mobile" which is a stripped down separate version of UIO intended for mobile devices, and we want to send users back to the same one they arrived from.